current/filters.md

```diff
@@ -133,10 +257,26 @@ val r = RecordLoader.loadArchives("example.warc.gz",sc)
 r.discardMimeTypes(mimetypes)
 ```
 
+### Scala DF
+
+```scala
+import io.archivesunleashed._
+import io.archivesunleashed.df._
+
+RecordLoader.loadArchives("example.warc.gz",sc)
+  .discardMimeTypesDF(Set("text/html"))
```

Comment:
This fails on the most recent build with 

```
<pastie>:36: error: value discardMimeTypesDF is not a member of org.apache.spark.rdd.RDD[io.archivesunleashed.ArchiveRecord]
possible cause: maybe a semicolon is missing before `value discardMimeTypesDF'?
          .discardMimeTypesDF(Set("text/html"))
           ^
```

Can be fixed by adding `.all()` or `.webpages()`

after the `RecordLoader` call